modules/serverless-uninstalling-knative-serving.adoc

```diff
@@ -1,27 +1,22 @@
 // Module included in the following assemblies:
 //
-// serverless/installing-openshift-serverless.adoc
+// serverless/installing_serverless/installing-openshift-serverless.adoc
 
 [id="uninstalling-knative-serving_{context}"]
 = Uninstalling Knative Serving
 
-To uninstall Knative Serving, you must remove its custom resource and delete the `knative-serving` namespace. 
-
-
-.Prerequisite
-
-* Installed Knative Serving
+To uninstall Knative Serving, you must remove its custom resource and delete the `knative-serving` namespace.
 
 .Procedure
 
-. To remove Knative Serving, use the following command:
+. Remove Knative Serving:
 +
 ----
-$ oc delete knativeserving.operator.knative.dev knative-serving -n knative-serving
+$ oc delete knativeservings.operator.knative.dev knative-serving -n knative-serving
 ----
 
-. After the command has completed and all pods have been removed from the `knative-serving` namespace, delete the namespace by using the command: 
+. Once the command has completed and all pods have been removed from the `knative-serving` namespace, delete the namespace:
```

Comment:
s/namespace, delete the namespace:/namespace, delete the namespace by entering the following command/